src/storage/config.rs

```diff
@@ -24,6 +24,10 @@ const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
 const DEFAULT_MAX_KEY_SIZE: usize = 8 * 1024;
 const DEFAULT_SCHED_CONCURRENCY: usize = 1024 * 512;
 const MAX_SCHED_CONCURRENCY: usize = 2 * 1024 * 1024;
+// The default GC life time is 10 minutes, we use 6 minutes as the default
+// scan lock smooth rate period, so that the scan lock can be smoothed in 6
+// minutes.
+const DEFAULT_SCAN_LOCK_SMOOTH_RATE_PERIOD: ReadableDuration = ReadableDuration::minutes(6);
```

Comment:
When there are multiple TiKV nodes, let's say 100 nodes, how much time GC can tolerate for the lock_scan stage?

Reply:
I think from GC's perspective the lock_scan should be as fast as it can. From TiKV hibernated regions perspective we should limit the wake up region numbers/s to a relevant smooth range like 1K regions/s.